src/metabase/internal_stats/users.clj

```diff
@@ -0,0 +1,23 @@
+(ns metabase.internal-stats.users
+  (:require
+   [metabase.db :as db]
+   [toucan2.core :as t2]))
+
+(defn email-domain-count
+  "Count all unique normalized domains found in active user emails"
+  []
+  (:count (t2/query-one {:select [[:%count.* :count]]
+                         :from [[{:select-distinct (condp contains? (db/db-type)
+                                                     #{:postgres}  [[[:split_part :email [:inline "@"] [:inline 2]]]]
+                                                     #{:h2 :mysql} [[[:substring :email [:locate "@" :email]]]])
+                                  :from [:core_user]
+                                  :where [:and
+                                          [:= :is_active true]
+                                          ;; Exclude @api-key because we probably don't want to count if it exists
+                                          ;; and it causes test flakes
+                                          [:not [:like :email [:inline "%@api-key.invalid"]]]]} :distinct_emails]]})))
```

Comment:
you could more easily use the type property
```sql
select id ,email, type from core_user where id in (13371338, 8, 2);
    id    |                               email                               |   type
----------+-------------------------------------------------------------------+----------
 13371338 | internal@metabase.com                                             | internal
        8 | user@metabase.com                                                 | personal
        2 | api-key-user-d86a47da-df1f-4395-a755-5c9b6f42e8cf@api-key.invalid | api-key
```

we only want `personal` type keys. Ignore internal and api-keys

Reply:
Probably want this for the ext-users stat too.